### PR TITLE
added create order and get order as well as their routes by: (the girls)

### PR DESCRIPTION
**Pull request by Iroh-Omolola**

### CODE REVIEW

File: src/controllers/ProductController.js [hunk 1]
```diff
@@ -120,6 +120,64 @@ const ProductController = {
         .json({ status: 'fail', message: 'server err', err });
     }
   },
+  // ORDER Controller===================
+createOrder: async (req, res) => {
+  const { userId, items } = req.body;
+  // if (!access || access !== 'admin') {
```
Comment (ibrahim013) [inline src/controllers/ProductController.js]: Delete codes, when you don't need them.

File: src/controllers/ProductController.js [hunk 2]
```diff
@@ -120,6 +120,64 @@ const ProductController = {
...
+  //   return res.status(401).json({ status: 'fail', message: 'unauthorized' });
+  // }
+  if (!userId || !items ) {
+    return res
+      .status(400)
+      .json({ status: 'fail', message: 'Please select a product' });
```
Comment (ibrahim013) [inline src/controllers/ProductController.js]: the response message should have been something like `product cart is empty`

**Review by ibrahim013 — COMMENTED** [review]
Some few comments

**Comment by Iroh-Omolola** [comment]
I've made changes but can't pull a request. kep saying "it's conflicting"

**Pull request merged**